demo/chbench/mzcompose.yml

```diff
@@ -96,6 +96,7 @@ services:
       KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka:9092,EXTERNAL://${KAFKA_HOST:-kafka}:9093
       KAFKA_METRIC_REPORTERS: "io.confluent.metrics.reporter.ConfluentMetricsReporter"
       KAFKA_BROKER_ID: 1
+      KAFKA_LOG_RETENTION_HOURS: 8760
```

Comment:
i think this is fine but just want to double check - will we run into a similar issue a year from now? if we don't want to fiddle with figuring out how to enable indefinite retention on this i think we could also just add two zeroes to this number and call it a day

Reply:
I was able to test "-1" and it works. I've updated the PR. Thanks!